src/app/firedancer/topology.c

```diff
@@ -462,6 +430,10 @@ fd_topo_initialize( config_t * config ) {
   fd_topo_tile_t * rpcserv_tile = NULL;
   if( enable_rpc ) rpcserv_tile =  fd_topob_tile( topo, "rpcsrv",  "rpcsrv",  "metric_in",  tile_to_cpu[ topo->tile_cnt ], 0,        0 );
 
+  fd_topo_tile_t * snaprd_tile  =  fd_topob_tile( topo, "SnapRd", "SnapRd", "metric_in", tile_to_cpu[ topo->tile_cnt ], 0, 0 );
```

Comment:
```suggestion
  fd_topo_tile_t * snaprd_tile  =  fd_topob_tile( topo, "snaprd", "SnapRd", "metric_in", tile_to_cpu[ topo->tile_cnt ], 0, 0 );
```

Please follow convention, consistency more important than having better names for a few things